core/src/main/java/com/graphhopper/routing/util/EncodingManager.java

```diff
@@ -510,6 +510,76 @@ public boolean canSkip() {
         }
     }
 
+    /**
+     * Determine whether a relation is routable for one of the added encoders.
+     *
+     * @return if at least one encoder consumes the specified relation. Additionally the specified acceptRelation is changed
+     * to provide more details.
+     */
+    public boolean acceptRelation(ReaderRelation relation, AcceptRelation acceptRelation) {
+        if (!acceptRelation.isEmpty())
+            throw new IllegalArgumentException("AcceptRelation must be empty");
+
+        for (AbstractFlagEncoder encoder : edgeEncoders) {
+            acceptRelation.put(encoder.toString(), encoder.getRelationAccept(relation));
+        }
+        return acceptRelation.hasAccepted();
+    }
+
+    public static class AcceptRelation {
```

Comment:
Can you explain what this class is needed for ?

Reply:
It is not needed.